### ISSUE TITLE
Remove gleam_stdlib dependency

### ISSUE THREAD

**Issue by inoas** [inoas/glychee#1]
Uses: `io.print`, `io.println`, `list.each`, `list.map`

Uses some more in the example file, which could be removed as well, the example can live on as documentation.

Once these are removed, glychee can be used to benchmark gleam_stdlib functions.

**Comment by inoas** [comment]
Removed gleam_erlang and gleeunit deps already:

https://github.com/inoas/glychee/tree/remove-stdlib-dep

**Comment by inoas** [comment]
Closed in 0.2.0.

Now even in gleam_stdlib you can do:

```sh
gleam add glychee
gleam touch src/benchmark.gleam
```

paste:

```gleam
if erlang {
  import glychee/benchmark
  import gleam/list
  import gleam/int

  pub fn main() {
    benchmark.run(
      [
        benchmark.Function(
          label: "list.sort()",
          fun: fn(test_data) { fn() { list.sort(test_data, int.compare) } },
        ),
      ],
      [
        benchmark.Data(label: "pre-sorted list", data: list.range(1, 100_000)),
        benchmark.Data(
          label: "reversed list",
          data: list.range(1, 100_000)
          |> list.reverse,
        ),
      ],
    )
  }
}
```

run:

```sh
gleam clean && \
gleam build && \
erl -pa ./build/dev/erlang/*/ebin -noshell -eval 'gleam@@main:run(benchmark)'
```
